dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -244,32 +244,32 @@ BEGIN
     EDITTEXT 14015, 58, 128, 170, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
     LTEXT "", -1, 8, 144, 221, 1, SS_ETCHEDHORZ | WS_CHILD | WS_VISIBLE
     LTEXT "Atribute:", 14020, 8, 152, 46, 10
-    AUTOCHECKBOX "Nem&odificabil", 14021, 58, 151, 68, 10
-    AUTOCHECKBOX "Asc&uns", 14022, 126, 151, 55, 10
-    AUTOCHECKBOX "Ar&hivabil", 14023, 181, 151, 59, 10
-    PUSHBUTTON "A&vansate…", 14028, 158, 170, 70, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP
+    AUTOCHECKBOX "&Doar în citire", 14021, 58, 151, 68, 10
+    AUTOCHECKBOX "&Ascuns", 14022, 126, 151, 55, 10
+    AUTOCHECKBOX "Ar&hivare", 14023, 181, 151, 59, 10
+    PUSHBUTTON "&Complex", 14028, 158, 170, 70, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP
```

Comment:
screenshot of those attributes-accels also please from 2k3sp2 romanian.

Reply:
![2024-03-27 00_47_56-Resource Hacker - shell32 dll mui](https://github.com/reactos/reactos/assets/113553683/911b448c-d45c-49e5-83f3-85acbf262177)

Reply:
So it must bee "Asc&uns" then, not "&Ascuns"!

Reply:
It's now I saw it. Now it's fixed!

Reply:
Resolved.

Reply:
I come back. It's weird, but true. Here it is "Asc&uns" and not "&Ascuns". It seems it is an exception. I can show you the proof.

![2024-04-06 23_17_32-](https://github.com/reactos/reactos/assets/113553683/2469673d-2aa1-4f9e-8145-f90465b53972)
![2024-04-06 23_17_54-Dialog - 1042](https://github.com/reactos/reactos/assets/113553683/6aa3ebde-a451-4b83-bece-23ecfeed974c)